app/views/mortgage_calculator/stamp_duties/_first_time_buyer_explanation.html.erb

```diff
@@ -0,0 +1,44 @@
+<p><%= I18n.t("stamp_duty.how_calculated_ftb", amount: ftb_threshold) %></p>
```

Comment:
TBH all the string in this file can be rendered with a single quote rather double quotes which is to be reserved for cases of string interpolation.